src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,20 @@
+package Duke.Task;
+
+import Duke.DukeException;
+
+public class Deadline extends Task{
+    protected String timing;
+
+    public Deadline(String description, String timing) throws DukeException {
+        super(description);
+        this.timing=timing;
+        if(description.isBlank()||timing.isBlank()){
+            throw new DukeException("DL incomplete");
```

Comment:
Perhaps you can replace the string literal into a constant to improve readability?